### DISCUSSION TITLE
h2-console problem on master branch

### DISCUSSION THREAD

**Discussion by guilhermecampello** [RADAR-base/RADAR-Docker#248] [General]
Hi! 

I'm moving the discussion from the slack channel to here as suggested by @yatharthranjan.

Firstly, thank you all for your help, and forgive me for annoying you so much 😝 .

As reported by PedroZuccolo, we are unable to access the Database from the menu on the ManagementPortal, it tries to access the url /h2-console and returns 404. 

Xi Bai gave a clear explanation about how to solve this problem but now I can't find those spring files. Where are they located?

Thank you!

**Comment by blootsvoets** [comment]
Note that by default, I don't think the h2 console should be accessible. The data is not stored in H2 by default but in a postgresql database. You can access that by running on the command-line in the radar-cp-hadoop-stack directory:
```bash
source <(sudo cat .env)
sudo bin/radar-docker exec radarbase-postgresql psql -h localhost -U $POSTGRES_USER managementportal
\dt
SELECT * FROM source_type;
Ctrl+D
```

**Comment by yatharthranjan** [comment]
HI, Can you also give a summary of what issue you have that prompted you to access the database?

From slack, I think you are just looking for data extracted from RADAR-base which is not in the database but on the filesystem. The path, where the data ends up, is defined by the property `RESTRUCTURE_OUTPUT_DIR` in `.env` file.

**Reply by guilhermecampello** [reply]
Hi, we are trying to access and extract the data collected from the paired phones. Can you please provide me the easiest way to do this? I've tried all the options from Data Extraction section from the README but any of them worked out.

A summary of what I've tried and the problems faced:
1) ![image](https://user-images.githubusercontent.com/43826374/119705385-116c9080-be2f-11eb-8fb6-d16314bfb042.png)
There's no data at the output folder and also the radar-output.service doesn't exists

2) ![image](https://user-images.githubusercontent.com/43826374/119702220-9786d800-be2b-11eb-8603-1239ee834a09.png)
I can't figure out the name of the hdfds file's. I've tried the directory listed at the .env file and any of them worked.

3) ![image](https://user-images.githubusercontent.com/43826374/119702357-c00ed200-be2b-11eb-881d-06b8e9ec335f.png)
bin/hdfs-restructure file doesn't exists.

**Reply by guilhermecampello** [reply]
Also, is the data stored at postgres?
I can't access it from outside the server because the container ports are not mapped for external access. I've mapped port 5432:5432 at the docker-compose file and it didn't solve. But maybe I should do a clean restart for it to work.

**Reply by guilhermecampello** [reply]
I'm currently running on master branch. Should I try pulling from another branch?

**Reply by yatharthranjan** [reply]
Hi @guilhermecampello, I apologise as the docs are not up to date. This is now an automated process and the steps you stated above are not needed. As I mentioned before, the data should be extracted on the path on the filesystem defined by the property RESTRUCTURE_OUTPUT_DIR in `.env` file. By default, this will be in `RADAR-Docker/dcompose-stack/radar-cp-hadoop-stack/output/`
